staging/src/k8s.io/component-base/metrics/prometheus/informer/metrics.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package informer
+
+import (
+	"k8s.io/client-go/tools/cache"
+	k8smetrics "k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+)
+
+const (
+	Subsystem                    = "informer"
+	StoreItemTotalKey            = "store_item_total"
+	QueuedItemTotalKey           = "queued_item_total"
+	PendingNotificationsTotalKey = "pending_notifications_total"
```

Comment:
The `_total` suffix is meant for counters only, I'd suggest going with:
```
stored_items
queued_items
pending_notifications
```

Reply:
Done.